### PR TITLE
Remove CSI client in Helm chart

### PR DESCRIPTION
**Pull request by ssz1997**
An example should not be included inside helm chart. Add to doc instead.

### CODE REVIEW

File: docs/en/deploy/Running-Alluxio-On-Kubernetes.md [hunk 1]
```diff
@@ -1504,12 +1504,29 @@ Here are some common properties that you can customize:
...
-to generate related templates. All CSI related templates will be under `${ALLUXIO_HOME}/integration/kubernetes/csi`.
-
 **Step 2: Deploy CSI services**
+You can use Helm to start the Alluxio CSI components with Alluxio cluster, or `kubectl` to create the resources manually,
+or parts from Helm and parts manually.
+
+{% navtabs deployCSI %}
+{% navtab helm %}
+
+To start the CSI components via helm chart, set the following values in your helm configuration file:
```
Comment (LuQQiu) [inline docs/en/deploy/Running-Alluxio-On-Kubernetes.md:1514]: any specific helm configuration file example in our doc?

File: docs/en/deploy/Running-Alluxio-On-Kubernetes.md [hunk 2]
```diff
@@ -1475,7 +1475,7 @@ Other than using Alluxio FUSE daemon, you could also use CSI to mount the Alluxi
 In order to use CSI, you need a Kubernetes cluster with version at least 1.17,
```
Comment (LuQQiu) [inline docs/en/deploy/Running-Alluxio-On-Kubernetes.md:1475]: For the CSI section, can you tell what's the differences between CSI and using helm/kubectl to deploy the Fuse daemon?

**Review by LuQQiu — CHANGES_REQUESTED** [review]
Thanks, left some questions

File: docs/en/deploy/Running-Alluxio-On-Kubernetes.md (hunk 1 above)
Comment (ssz1997) [reply docs/en/deploy/Running-Alluxio-On-Kubernetes.md:1514]: Our helm chart provides a set of default configurations

File: docs/en/deploy/Running-Alluxio-On-Kubernetes.md [hunk 3]
```diff
@@ -1323,7 +1323,8 @@ application containers can simply mount the Alluxio FileSystem.
 
 #### FUSE daemon
 
-One way to use the POSIX API is to deploy the Alluxio FUSE daemon.
+One way to use the POSIX API is to deploy the Alluxio FUSE daemon, creating pods running Alluxio Fuse processes inside
```
Comment (LuQQiu) [inline docs/en/deploy/Running-Alluxio-On-Kubernetes.md:1326]: inside?

File: docs/en/deploy/Running-Alluxio-On-Kubernetes.md [hunk 4]
```diff
@@ -1471,11 +1472,14 @@ spec:
 
 #### CSI
 Other than using Alluxio FUSE daemon, you could also use CSI to mount the Alluxio FileSystem into application containers.
+The pods running Alluxio Fuse processes will only be launched when the application pods ask for data from Alluxio through CSI.
+A Fuse pod will be terminated when no pod utilizes the Fuse pod. In other words, the Fuse process exists only when it's needed,
```
Comment (LuQQiu) [inline docs/en/deploy/Running-Alluxio-On-Kubernetes.md:1476]: when no application pod requests data through the Fuse path? Fuse path or Fuse mounted volume or how to describe this part?

Comment (LuQQiu) [reply docs/en/deploy/Running-Alluxio-On-Kubernetes.md:1476]: when it's needed or when it's being used?

**Review by LuQQiu — COMMENTED** [review]
left some minor comments

File: docs/en/deploy/Running-Alluxio-On-Kubernetes.md (hunk 3 above)
Comment (ssz1997) [reply docs/en/deploy/Running-Alluxio-On-Kubernetes.md:1326]: I meant the processes run inside the pods. Removed.

**Comment by ssz1997** [comment]
@LuQQiu PTAL. Thanks!

**Review by LuQQiu — APPROVED** [review]

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**